data/property-sort-orders/smacss.txt

```diff
@@ -1,15 +1,23 @@
 # SMACSS Property Order
 # http://smacss.com/book/formatting
 
+content
```

Comment:
I think it would be a good idea to precede this with a comment about these properties being outside of the groups listed at https://smacss.com/book/formatting#grouping (sine the following rules are grouped by a comment “heading”)